plugin/src/test/java/org/opensearch/ml/rest/RestMLDeployModelActionIT.java

```diff
@@ -34,6 +36,9 @@ public void setup() throws IOException {
     }
 
     public void testReDeployModel() throws InterruptedException, IOException {
+        // Skip test if running on Mac OS, https://github.com/opensearch-project/ml-commons/issues/844
+        Assume.assumeFalse(System.getProperty("os.name").startsWith("Mac OS X"));
```

Comment:
What do you think about adding this in the Setup? For mac, if we are ignoring the test, then may be we don't need to register the model group and subsequent lines in Setup?

Reply:
There could another test which could still run, but for now I agree it makes sense.